### DISCUSSION TITLE
Issue in Keycloak

### DISCUSSION THREAD

**Discussion by arya44mishra** [keycloak/keycloak#29437] [Q&A]
Hi Team,
Good Morning
I have created realm name (SSO-QA) for qa Env. It gets automatically deleted after a week ,only one realm is left - master realm.
Please help me out in this issue urgently.

**Comment by codespearhead** [comment]
I'm almost certain that the problem doesn't lie in Keycloak.

Chances are your team is deleting the realm or the Keycloak container is being recreated from scratch every week.

The Keycloak logs will tell you what happened.

**Reply by arya44mishra** [reply]
Thanks For responding @codespearhead 
But I am very well sure ,No one is deleting.Only I have the credentials for it .
No Logs are there , i had checked that also.

**Reply by codespearhead** [reply]
If no logs are there, then its not that the other realm is being deleted: the entire container is being recreated from scratch.

How and where are you deploying Keycloak?

**Reply by arya44mishra** [reply]
deploying my keycloak on server , only master realm left.

**Reply by codespearhead** [reply]
By my last question I mean what quickstart from the [Getting Started section](https://www.keycloak.org/guides) you followed and what production environment you deployed it to (AWS, VMWare ESXi, Openshift, bare metal, etc).
